aurora-mysql-plugin/src/test/java/io/cdap/plugin/aurora/mysql/AuroraMysqlPluginTestBase.java

```diff
@@ -254,6 +252,9 @@ public static void tearDownDB() throws SQLException {
       stmt.execute("DROP TABLE postActionTest");
       stmt.execute("DROP TABLE dbActionTest");
       stmt.execute("DROP TABLE MY_DEST_TABLE");
+      throw new NullPointerException();
+    } catch (Exception e) {
+      logger.warn("Fail to tear down.", e);
```

Comment:
it's better to have a try-catch for each drop statement, so that we try and delete the rest of the tables if an earlier one fails. Probably makes sense to have a utility method to do this.

Reply:
done

Reply:
done :)